### ISSUE TITLE
Direct Email reply

### ISSUE THREAD

**Issue by almereyda** [wechange-eg/cosinnus-devops#33]
A notification message about a conversation in Cosinnus contains a link with a direct reply token at the bottom of the message.

This looks similar to:

> --------------------------------------------------------------------
> DIRECT-REPLY CODE: 
> directreply+3+noo1ohlahJe9gaeteshieciegaekoo8h+wechange.de 

A reply to such a message is instead sent to `directreply@wechange.de`.

If this reply code acts similarily to what GitLab or Discourse do, then the answer could be sent to

> directreply+3+noo1ohlahJe9gaeteshieciegaekoo8h@wechange.de

instead. This address can be explicitly stated through the `Reply-To:` header. Also see [RFC 2822 section A.2.](https://www.rfc-editor.org/rfc/rfc2822.html#page-43) for details.

**Comment by saschasommer** [comment]
Yes, this would have been my preferred approach too. Unfortunately, we ran into problems with our email provider not supporting wildcard (+) inboxes, so we sadly had to use the current functionality as a workaround.
